config/src/config/storage_config.rs

```diff
@@ -118,6 +120,12 @@ pub struct StoragePrunerConfig {
     pub state_store_pruning_batch_size: usize,
     /// The offset for user pruning window to adjust
     pub user_pruning_window_offset: u64,
+    /// The pruning interval of state pruner. The state pruner will sleep for this period of time
+    /// after pruning each batch.
+    pub state_pruner_time_interval_in_ms: u64,
+    /// The pruning interval of ledger pruner. The ledger pruner will sleep for this period of time
+    /// after pruning each batch.
+    pub ledger_pruner_time_interval_in_ms: u64,
 }
```

Comment:
Separately, does it make sense to make configs for both pruners separate? Would have avoided issue happened yesterday.

```
struct PrunerConfig {
    ledger_pruner_config: LedgerPrunerConfig,
    state_merkle_pruner_config: StatePrunerConfig, 
}
```

also, notice the term "pruner_store" is not proper, let's use "state_merkle_pruner"

probably address in a separate PR.

Reply:
Yes I am planning to do that in an other PR.